webui-page/webui.js

```diff
@@ -491,7 +509,7 @@ function setupNotification() {
 }
 
 status();
-setInterval(function(){status();}, 1000);
+setInterval(function(){status();}, 10000);
```

Comment:
With a timeout of `1000`, one can see the seconds counting in the webui. IMO this gives a much more "real-time" feel of the app. Please revert.